### PR TITLE
Slp5 update

### PR DESCRIPTION
**Pull request by Dhanushkas**
## Automation tests
 - Unit tests 
   > Test each record type CURD operations
 - Integration tests
   > 

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: src/netsuite/recordtypes/writable_records.bal [hunk 1]
```diff
@@ -250,13 +316,30 @@ public type Opportunity record {
...
+    decimal overallbalance?;
+    decimal overallunbilledorders?;
+    decimal rangeHigh?;
+    decimal rangeLow?;
+    decimal total?;
+    decimal totalCostEstimate?;
+    decimal weightedTotal?;
+    decimal primarycurrency?;
+    decimal primarycurrencyfxrate?;
+
```
Comment (chamil321) [inline src/netsuite/recordtypes/writable_records.bal:341]: Remove new lines

Comment (chamil321) [reply src/netsuite/recordtypes/writable_records.bal:341]: Check other places as well

File: src/netsuite/tests/main_test.bal [hunk 2]
```diff
@@ -63,8 +63,8 @@ function testCustomizedCompanySpecificRecord() {
         custrecord_address: "Col 3"
     };
 
-    string createdId = createOrSearchIfExist(customRecord, "name IS ballerina testing", customPath);
-    customRecord = <BalTestCustomRecord> readRecord(<@untainted> createdId, BalTestCustomRecord);
+    string createdId = createOrSearchIfExist(customRecord, "name IS  Bal Test Custom Record", customPath);
+    customRecord = <BalTestCustomRecord> readRecord(<@untainted> createdId, BalTestCustomRecord,customPath);
```
Comment (chamil321) [inline src/netsuite/tests/main_test.bal:67]:
```suggestion
    customRecord = <BalTestCustomRecord> readRecord(<@untainted> createdId, BalTestCustomRecord, customPath);
```

File: src/netsuite/netsuite_constants.bal [hunk 3]
```diff
@@ -99,7 +99,9 @@ const string RECORD_PATH_ADDRESSBOOK_ADDRESS = "/addressbookaddress";
 const string RECORD_PATH_SHIPPING_ADDRESS = "/shippingAddress";
 const string RECORD_PATH_BILLING_ADDRESS = "/billingaddress";
 const string RECORD_PATH_CURRENCY = "/currency";
-const string RECORD_PATH_NON_INVENTORY_ITEM = "/nonInventoryItem";
+//-------SLP4----------Should change into 3 types of Sales,ReSale etc.
```
[the remark this answers is not in the document]
Comment (Dhanushkas) [reply src/netsuite/netsuite_constants.bal:102]: [API issue created](https://github.com/ballerina-platform/module-ballerinax-netsuite/issues/8#issue-739297453)

File: src/netsuite/recordtypes/writable_records.bal (hunk 1 above)
Comment (Dhanushkas) [reply src/netsuite/recordtypes/writable_records.bal]: Done

**Review by kasun04 — APPROVED** [review]

**Pull request merged**